src/main/java/mate/academy/dao/impl/BookDaoImpl.java

```diff
@@ -0,0 +1,110 @@
+package mate.academy.dao.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Book;
+import mate.academy.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+@Dao
+public class BookDaoImpl implements BookDao {
+    private static final int TITLE_INDEX = 1;
+    private static final int PRICE_INDEX = 2;
+    private static final int ID_INDEX = 3;
+    private static final String INSERT_REQUEST = "INSERT INTO books(title, price) VALUES(?, ?)";
+    private static final String REQUEST_BOOK_FROM_DB = "SELECT * FROM books WHERE id = ?";
+    private static final String GET_ALL_REQUEST_BOOK = "SELECT * FROM books";
+    private static final String UPDATE_REQUEST = "UPDATE books SET title = ?, price = ? WHERE id = ?";
+    private static final String DELETE_BOOK_REQUEST = "DELETE FROM books WHERE id = ?";
```

Comment:
I'd recommend you to put this queries in the methods where they are used in the way of good practice 
![image](https://github.com/mate-academy/jv-jdbc-intro/assets/32339836/18b0418d-c61e-4e5d-923c-20cfd92a18de)
It will be hard to menage them, if you have a lot of queries and with a lot of condition and sub-queries